kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverVariantConfig.kt

```diff
@@ -216,4 +216,21 @@ public interface KoverVariantCreateConfig: KoverVariantConfig {
      * If [optional] is `true` and a variant with given name is not found in the current project - in this case, the variant will not be searched even in dependencies.
      */
     public fun addWithDependencies(vararg variantNames: String, optional: Boolean = false)
+    /**
+     * Add to created variant classes, tests and instrumented classes from report variant with name [variantNames].
+     * This variant is taken only from the current project.
```

Comment:
```suggestion
     * These variants are taken only from the current project.
```
Given that `Iterable` is usually some kind of collection, using plural form seems more logical